File: problems/prob03/prob3.js
function createList(num) {
  list = [];
  for (var i = 1; i <= num; i++) {
    list.push(Math.floor((Math.random() * 100)));
  }
  return list;
}

function getMax(list) {
  return Math.max.apply(Math, list);
}

function getMin(list) {
  return Math.min.apply(Math, list);
}

var list = createList(20);
console.log("The numbers are " + list);

var maxNumber = getMax(list);
console.log("The max number is " + maxNumber);

var minNumber = getMin(list);
console.log("The min number is " + minNumber);

File: problems/prob03/prob3.py
from random import randrange


def rand_n(num):
    lst = []
    for i in range(num):
        lst.append(randrange(100))
    return lst


def max_n(lst):
    return max(lst)


def min_n(lst):
    return min(lst)


lst = rand_n(20)
print("The numbers are " + str(lst)[1:-1])

max_num = max_n(lst)
print("The max number is " + str(max_num))

min_num = min_n(lst)
print("The min number is " + str(min_num))
